doc/api/net.md

```diff
@@ -174,6 +268,12 @@ server.listen({
 });
 ```
 
+The `server.listen()` method is asynchronous. When the server has been bound,
+the [`'listening'`][] event will be emitted.
+
+If a `callback` function is given, it will be added as a listener for the
```

Comment:
"given", "provided", "specified", "passed" - so many synonyms we use :-(. Not specific to this PR, but do you have a favorite that we should push to use across the API docs?